tasks/18_humanize-format/humanize-format.js

```diff
@@ -0,0 +1,19 @@
+function humanizeFormat(number) {
+  if (!number) {
```

Comment:
In this case for 0 you will return empty string as well

Reply:
Despite such case isn't mentioned in tech task, fixed